assets/prometheus-k8s/rules.yaml

```diff
@@ -380,6 +380,13 @@ spec:
       record: node_role_os_version_machine:cpu_capacity_sockets:sum
     - expr: clamp_max(sum(alertmanager_notifications_total),1)
       record: cluster:alertmanager_routing_enabled:max
+    - alert: ClusterOOMWarning
```

Comment:
Hmm not sure this is related to the cluster 100% of the time, could be just wrong memory specifications, maybe something like "KubePodOOMWarning" naming would be better.

Reply:
Thank you for the great comments. Updated.